### DISCUSSION TITLE
Intercom plugin (push notifications)

### DISCUSSION THREAD

**Discussion by StevenH86** [ionic-team/capacitor#3667] [Q&A]
Hi all,

I'm trying to implement the intercom community plugin. Works well, but push does not work. The instructions say you just need to register for push notifications with the standard capacitor plugin, but it doesn't work and when I search the intercom plugin there is nothing in the code to handle push tokens and pass them on to intercom. 

Anyone able to help with this? It seems the plugin hasn't been updated for quite a while. 

I considered just using the cordova plugin, but not entirely sure how to implement it properly without being able to add values to the config.xml like you normally would. 

Cheers

**Comment by victorgutierrez** [comment]
I'm with the same problem.

I use capacitor.

**Reply by therealsquidgee** [reply]
Did you ever find a solution?
